src/scripts/fix_workspace_info.py

```diff
@@ -0,0 +1,85 @@
+"""
+Fixes workspace info to do the following.
+1. Make sure the "narrative" metadata field contains an int that points to the narrative.
+2. Make sure the "narrative_nice_name" metadata field is correct.
+3. Make sure the "is_temporary" metadata field exists and is correct.
+4. Adds a count of the number of narrative cells.
```

Comment:
Presumably this will also need to be updated on narrative save, right?

Reply:
Yup, we'll need another PR to the narrative repo.
Just wanted to throw this out there before adding extra work. But I'll make this a checkbox in the todos.